refurb/visitor/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .mapping import METHOD_NODE_MAPPINGS
+from .visitor import RefurbVisitor
+
+__all__ = ["METHOD_NODE_MAPPINGS", "RefurbVisitor"]
```

Comment:
Happy to make the change, but hear this first :)

I have always used lists because the differences are not significant in terms of performance or readability but there is a somewhat-philosophical argument for lists, it's captured very well here:
https://stackoverflow.com/a/66098991/116957

What do you think? :)

Reply:
I am fine with it either way. I am just more used to seeing it as a tuple is all. I guess a list would be nice if you plan on dynamically modifying it (like one of the answers mentions), but if you know "these are the only fields I want to export" then a tuple might be better suited.